identity-field/README.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Identity Field
+category: Structural
+language: en
+tag:
+- Enterprise Integration Pattern
+---
+
+## Also known as
+
+> P of EAA
+
+## Intent
+
+> Saves a database ID field in an object to maintain an identity between an in-memory object and a database row.
+
+## Explanation
+
+>The Identity Field design pattern is a structural design pattern that allows you to maintain the identity of an object between the application and the database. It is also known as P of EAA, which stands for Primary Key of Entity-Attribute-Association.
+
+>The Identity Field design pattern works by adding a field to your object that matches the primary key of the corresponding database table. This field is then used to identify the object in the database.
```

Comment:
Explanation chapter has a standard format we are using. See here: https://github.com/iluwatar/java-design-patterns/wiki/02.-Pattern-template Also check the published patterns for examples.